### PR TITLE
Adding evermore

### PR DESCRIPTION
**Pull request by pfackeldey**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/evermore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/evermore-{{ version }}.tar.gz
+  sha256: a56b6321f0ed22c1eaeb2679baf2d7b12d0c9c8374486e46b11b97cfd4f6c2c8
+
+build:
+  noarch: python
```
Comment (pfackeldey) [inline recipes/evermore/meta.yaml:13]: Windows builds failing are expected as JAX is not supported on this platform. If my understanding is correct it's ok to leave failing windows builds on "noarch".

**Comment by pfackeldey** [comment]
@conda-forge/help-python, ready for review!

**Comment by matthewfeickert** [comment]
> @conda-forge/help-python, ready for review!

I don't remember if they can see this or not from first time conda-forge recipe contributors, so I'm tagging @conda-forge/help-python again to make sure this gets added to the review list. :+1:

File: recipes/evermore/meta.yaml (hunk 1 above)
Comment (matthewfeickert) [reply recipes/evermore/meta.yaml:13]: I told @pfackeldey this given https://github.com/conda-forge/staged-recipes/pull/26293#discussion_r1594838679.

**Pull request merged**